en/developer_guide/block_display.rst

```diff
@@ -0,0 +1,95 @@
+Block display
+=============
+
+Display Independence
+--------------------
+
+In Front Office, some blocks need specific stylesheets and javascript file.
+To perform this, the main node layout loads the requireJS library which is a JavaScript file and module loader
+(see `requireJS <http://requirejs.org/>`_).
```

Comment:
the link to the requireJs page should be on the first time you give the techno name